src/configuration.ts

```diff
@@ -44,14 +45,15 @@ export class RLSConfiguration {
      * directory.
      */
     public readonly rlsRoot: string | null;
-
+    
     public static loadFromWorkspace(): RLSConfiguration {
         const configuration = workspace.getConfiguration();
 
         return new RLSConfiguration(configuration);
     }
 
     private constructor(configuration: WorkspaceConfiguration) {
+        this.rustupPath = configuration.get('rustup.path', 'rustup');
```

Comment:
Could you use something like`rust-client.rustupPath` for the configuration name please? Could you also add the option to the `configuration` section of package.json.

Reply:
Yeah, you are right.

I will submit patch when I get home.